src/components/commons/WarnDescription.tsx

```diff
@@ -19,11 +19,13 @@ const WarnDescription = (props: LimitWarnPropType) => {
 export default WarnDescription;
 
 const Wrapper = styled.div<{ $isShown: boolean }>`
-  display: ${({ $isShown }) => ($isShown ? 'flex' : 'none')};
+  display: flex;
   gap: 0.5rem;
   align-items: center;
 
   width: 100%;
+
+  opacity: ${({ $isShown }) => !$isShown && '0'};
```

Comment:
```suggestion
  opacity: ${({ $isShown }) => $isShown || '0'};
```
p5) false 값일 때 '0'을 지정해주고 싶다면, && 말구 || 활용해주는 방법도 있어요!

Reply:
앗 넹 !! 감사합니당 !!